test/test_helper.rb

```diff
@@ -0,0 +1,12 @@
+require "simplecov"
```

Comment:
I *think* we need
`SimpleCov.start`
here, too.

Reply:
I need to get confirmation, but test_helper files are generally reserved for third party requires, but the file directory paths generally remain in their respective files so that it is clear what is being called when you are looking at the code.

To @kristastadler point, we do need to add that snippet here as well...